whylog/rulesbase/__init__.py

```diff
@@ -1,6 +1,87 @@
+import re
+import yaml
+from abc import ABCMeta, abstractmethod
 
+class AbstractConfigAccess(object):
```

Comment:
I'm not sure if the name Config is correct here. In my opinion the word Base is better here. Let talk about it.